### ISSUE TITLE
space after class definition not detected

### ISSUE THREAD

**Issue by spaceone** [PyCQA/pycodestyle#590]
```
printf 'class Foo (Bar):\n    pass\n' | pycodestyle -
                 ^
```
With pycodestyle 2.0.0 and 2.1.0.dev0.

**Comment by sigmavirus24** [comment]
So there's nothing in PEP-0008 that precludes this. The only thing related to this is in [Pet Peeves](https://www.python.org/dev/peps/pep-0008/#pet-peeves) that describes calling a function like so:

```py
spam (1)
```

As opposed to

```py
spam(1)
```

I'm inclined to say that this is not a bug.

**Comment by spaceone** [comment]
A little more agressive example:
```
printf 'class Foo \t (Bar):\n    pass\n' | pycodestyle -
```

is pycodestyle about bad coding style or implementing PEP8 strictly? This code looks very ugly.

**Comment by sigmavirus24** [comment]
> is pycodestyle about bad coding style or implementing PEP8 strictly?

pycodestyle is what used to be called pep8 and is very firmly rooted in the document's guidance.

**Comment by spaceone** [comment]
At least my second example needs to be detected as E101 mixed-spaces-and-tabs, right?

**Comment by sigmavirus24** [comment]
I believe E101 is purely about indentation, but I haven't double-checked that.

**Comment by spaceone** [comment]
Yes, E101 tells about indentation as all E1 are indentation issues. Nevermind, it is up to you what you are doing with this issue. My opinion is that it is not preferable style. Maybe one needs to discuss this at PEP8 directly or maybe pycodestyle can just introduce some new E2 code. Just wanted to let you know this issue as I saw it in our code.

**Comment by lathama** [comment]
If this is a non-issue now can you close this spaceone?

**Comment by sigmavirus24** [comment]
@lathama I'm not sure this is a non-issue.

**Comment by lathama** [comment]
maybe edit the title then.

**Comment by sigmavirus24** [comment]
Why, the title is still accurate? A space after a class definition is not detected and ... _maybe_ it should be.

**Comment by lathama** [comment]
Sorry, attempting to grok project status. Ack on the maybe, a tag or label identifying feature request would be good here.
